components/dash-core-components/tests/integration/input/test_input_basics.py

```diff
@@ -65,6 +69,12 @@ def test_inbs002_user_class(dash_dcc):
     dash_dcc.start_server(app)
 
     dash_dcc.find_element(".test-input-css")
-    dash_dcc.percy_snapshot("dcc styled input - width: 100%, border-color: hotpink")
+
+    def get_style(attr):
+        sel = "document.querySelector('.test-input-css input')"
+        return dash_dcc.driver.execute_script(f"return getComputedStyle({sel}).{attr}")
+
```

Comment:
There is `wait_for_style` with the browser or is that because we don't want to wait?

Reply:
Good call, I had forgotten about that one -> https://github.com/plotly/dash/pull/2069/commits/4d869c2cc693f88ff22c5e2e013bd9d83397ad3f

Reply:
The style test fails, I think the colors may need the alpha value.